samples/c2pa/plugin-dash/c2pa-plugin.js

```diff
@@ -73,26 +76,36 @@ async function c2pa_init(player, onPlaybackTimeUpdated) {
 
             let segs = tree[tag].search([e.time, e.time + 0.01]);
 
-            if (segs.length != 1)
+            if (segs.length > 1)
                 detail['error'] = 'Retrieved unexpected number of segments: ' + segs.length + ' for media type ' + type;
```

Comment:
Should we do the same as below when `seg.length>1`? (set status to undefined and continue)

Reply:
Good question... Maybe you have a better idea of when multiple segments can be retrieved instead of one? Based on that, we can make a decision

Reply:
Changed in latest commit